src/Radio.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { View, TouchableOpacity, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+// G A L I O - D E P E N D E N C Y
+import { Text } from '.';
+import GalioTheme, { withGalio } from './theme';
+
+class Radio extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: props.initialValue,
+    };
+
+    this.spaceAround = this.spaceAround.bind(this);
+  }
+
+  // A D D I N G - R E Q U I R E D - S P A C E (S) - B A S E D - O N - F L E X - D I R E C T I O N
+  spaceAround(direction) {
+    switch (direction) {
+      case 'row-reverse':
+        return { marginRight: 10 };
+      case 'column':
+        return { marginTop: 10 };
+      case 'column-reverse':
+        return { marginBottom: 10 };
+      default:
+        return { marginLeft: 10 };
+    }
+  }
+
+  // R E N D E R - L A B E L
+  renderLabel() {
+    const { label, disabled, flexDirection, labelStyle, styles } = this.props;
+
+    const labelStyles = [
+      styles.textStyles,
+      disabled && styles.disabledLabel,
+      labelStyle,
+      flexDirection && this.spaceAround(flexDirection),
+    ];
+
+    if (label) {
+      return <Text style={labelStyles}>{label}</Text>;
+    } else {
+      return null;
+    }
+  }
+
+  // O N P R E S S - H A N D L E R
+  radioPressHandler() {
+    this.setState({ checked: !this.state.checked }, () => this.props.onChange(this.state.checked));
+  }
+
+  render() {
+    const { props, state } = this;
+    const {
+      styles,
+      disabled,
```

Comment:
the `disabled` prop doesn't change the color of the radio circle to gray, this should change the color as well, not only the text

Reply:
oh that's true and noted. Thanks for the review